frontend/src/components/navigation/Navigation.tsx

```diff
@@ -0,0 +1,87 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useLocalization } from "@fluent/react";
+import styles from "./Navigation.module.scss";
+import { useIsLoggedIn } from "../../hooks/session";
+import { SignUpButton } from "./SignUpButton";
+import { SignInButton } from "./SignInButton";
+import { useProfiles } from "../../hooks/api/profile";
+import { UpgradeButton } from "./UpgradeButton";
+import { WhatsNewMenu } from "../layout/whatsnew/WhatsNewMenu";
+import { UserMenu } from "./UserMenu";
+import { AppPicker } from "./AppPicker";
+import { MenuToggle } from "./MenuToggle";
+
+/** Switch between the different pages of the Relay website. */
+export const Navigation = ({ ...props }) => {
+  const { l10n } = useLocalization();
+  const router = useRouter();
+  const isLoggedIn = useIsLoggedIn();
+  const profiles = useProfiles();
+  const homePath = isLoggedIn ? "/accounts/profile" : "/";
+  const hasPremium: boolean = profiles.data?.[0].has_premium || false;
+  const { theme, mobileMenuState, handleToggle } = props;
+
+  const ToggleButton = () => (
+    <a
+      href="#"
```

Comment:
A link that's not going anywhere should be a button - then you can also toggle it with e.g. the spacebar.